_posts/2023-06-29-101-code.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Robots 101 - The Code
+---
+
+Welcome to another blog post in our 101 series! Today, we're going to delve into
+the brains of your robot - its code.
+
+## Teamwork Makes The Code Work
+
+First and foremost, your robot's code should never be a one-person show. Make
+sure more than one person has access to the codebase and understands it. This is
+crucial for two reasons: First, if the primary coder is unavailable for any
+reason, others can step in and keep things moving. Second, multiple perspectives
+can identify potential issues and generate creative solutions that a single mind
+might overlook. Think of it as having a built-in code review process!
+
+## The Pulse of Your Robot - The Control Loop
+
+Your robot interacts with the environment in a continuous cycle of sensing and
+acting. It's essential to have a solid control loop in your code where your
+robot senses its environment and then decides what to do next based on that
+information. This loop should be at the core of your robot's operation, allowing
+it to adapt and react to its surroundings.
+
+## Backup Your Code: The Magic of Version Control Systems
+
+Mistakes and accidents happen, files get lost, and computers crash. To protect
+your work from these unpredictable events, ensure you regularly back up your
+code. Better yet, use a Version Control System (VCS) like Git. A VCS not only
```

Comment:
Is a specific mention of github itself worth it? I'd only heard of github and not git specifically before SR

Reply:
```suggestion
code. Better yet, use a Version Control System (VCS) like Git with GitHub. A VCS not only
```

Reply:
```suggestion
code. Better yet, use a Version Control System (VCS) like Git with GitHub. A VCS not only
```